CMakeLists.txt

```diff
@@ -61,21 +61,35 @@ endif()
 # Link against compressonator libs & set runtime lib
 if (UNIX)
 	if (VTFLIB_SHARED)
-		target_link_directories(vtflib PUBLIC thirdparty/lib/x64)
-		target_link_libraries(vtflib PUBLIC CMP_Compressonator pthread)
+		target_link_directories(vtflib PUBLIC thirdparty/lib/linux_x86_64)
+		target_link_libraries(vtflib PUBLIC
+				Compressonator
+				CMP_Core
+				CMP_Core_AVX
+				CMP_Core_AVX512
+				CMP_Core_SSE
+				pthread
```

Comment:
use cmake's `Threads::Threads` target instead

from stackoverflow:
```cmake
set(CMAKE_THREAD_PREFER_PTHREAD ON)
set(THREADS_PREFER_PTHREAD_FLAG ON)
find_package(Threads REQUIRED)

add_executable(test test.cpp)
target_link_libraries(test Threads::Threads)
```

also, is this actually required? i cant remember any vtflib code that uses pthreads or any threading stuff in `std`